src/test/scala/com/exasol/spark/DefaultSourceSuite.scala

```diff
@@ -17,4 +18,13 @@ class DefaultSourceSuite extends FunSuite with Matchers with MockitoSugar {
       thrown.getMessage === "A sql query string should be specified when loading from Exasol"
     )
   }
+
+  // scalastyle: off
```

Comment:
Is this `scalastyle:off` needed?

If so, could you please add identifier after off, for example, 
```scala
// scalastyle:off println
...
println("string")
// scalastyle:on
```

Additionally, you can also make it warning in `project/scalastyle-test-config.xml`.